src/main/java/fruitshop/strategy/SupplyOperation.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    private final Storage storage = new Storage();
```

Comment:
According to the common mistakes checklist, we should not begin class or method implementation with an empty line. The empty line after the class declaration on line 6 should be removed.